apis/druid/v1alpha1/druid_types.go

```diff
@@ -241,6 +241,12 @@ type DruidSpec struct {
 	// +kubebuilder:default:=true
 	RollingDeploy bool `json:"rollingDeploy"`
 
+	// DefaultProbes If set to true this will add default probes (liveness / readiness / startup) for all druid components
+	// but it won't override existing probes
+	// +optional
+	// +kubebuilder:default:=false
```

Comment:
I think it should be the default unless people want to disable it. Because it should work best "out of the box"